### PR TITLE
TC_004 and TC_005 are added

### PR DESCRIPTION
**Pull request by Natalia-Java-QA**
TC_004 and TC_005 are added honoring the BaseTest class

### CODE REVIEW

**Review by Maria7725 — APPROVED** [review]

File: src/test/java/YakuninaTest.java [hunk 1]
```diff
@@ -1,35 +1,34 @@
...
-        String value = message.getText();
-        assertEquals("Received!", value);
-
-        driver.quit();
+    //TC_004
+    @Test
+    public void testAccountButtonOpensLoginPage() {
+        String title = getDriver().getTitle();
+        assertEquals(title, "AskOmDch – Become a Selenium automation expert!");
+        getDriver().manage().timeouts().implicitlyWait(Duration.ofMillis(500));
```
Comment (virchi007) [inline src/test/java/YakuninaTest.java:15]: We already have in a baseTest an implicit wait. You probably can skip that line.

File: src/test/java/YakuninaTest.java [hunk 2]
```diff
@@ -1,35 +1,34 @@
...
+        assertEquals("https://askomdch.com/account/", getDriver().getCurrentUrl());
+        getDriver().manage().timeouts().implicitlyWait(Duration.ofMillis(500));
+    }
 
+    //TC_005
+    @Test
+    public void testShopNowButtonOpensUpStorePage() {
+        String title = getDriver().getTitle();
+        assertEquals(title, "AskOmDch – Become a Selenium automation expert!");
+        getDriver().manage().timeouts().implicitlyWait(Duration.ofMillis(500));
```
Comment (virchi007) [inline src/test/java/YakuninaTest.java:27]: Same here about implicit wait.

**Review by virchi007 — COMMENTED** [review]
I left a few comments in your code.

**Comment by tymocoder** [comment]
Good practice is to set the implicit wait time at the beginning of your test, before interacting with any elements on the page. This ensures that the implicit wait time is in effect for the entire test and avoids any unexpected behavior.

**Comment by tymocoder** [comment]
More than one assert in one test is not good.

File: src/test/java/YakuninaTest.java (hunk 1 above)
Comment (Natalia-Java-QA) [reply src/test/java/YakuninaTest.java:15]: thank you for your comment! I'll fix that in my next PR

File: src/test/java/YakuninaTest.java (hunk 2 above)
Comment (Natalia-Java-QA) [reply src/test/java/YakuninaTest.java:27]: same here :)

**Pull request merged**